src/relative-time/format.js

```diff
@@ -0,0 +1,35 @@
+define([
+	"../common/format-message"
+], function( formatMessage ) {
+
+/**
+ * format( value, numberFormatter, pluralGenerator, properties )
+ *
+ * @value [Number] The number to format
+ *
+ * @numberFormatter [String] A numberFormatter from Globalize.numberFormatter
+ *
+ * @pluralGenerator [String] A pluralGenerator from Globalize.pluralGenerator
+ *
+ * @properties [Object] containing relative time plural message.
+ *
+ * Format relative time.
+ */
+return function( value, numberFormatter, pluralGenerator, properties ) {
+
+	var relativeTime, message = properties[ "relative-type-" + value ];
```

Comment:
`message =` needs to go in its own line http://contribute.jquery.org/style-guide/js/#assignments